fvm/transaction_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
+const TopShotContractAddress = "0b2a3299cc857e29"
```

Comment:
Doesn't seem to be used. I don't think we should check for this address in particular